### PR TITLE
Android create account

### PR DESCRIPTION
**Pull request by lar-ryan-nick**
* Added register account activity
* Added functionality to switch between Login and register activities
* Changed default activity to register activity
* changed name of previous activity from MainActivity to LoginActivity to improve readability
* Screenshots:
screenshot_1517619099
screenshot_1517619103
screenshot_1517619110
screenshot_1517619135
screenshot_1517619143
screenshot_1517619146

### CODE REVIEW

File: android/.idea/markdown-exported-files.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pmh192) [inline android/.idea/markdown-exported-files.xml]: Is the .idea folder supposed to be committed?

File: android/app/src/main/java/com/tourneynizer/tourneynizer/LoginActivity.java [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.activity_login);
         View loginButton = findViewById(R.id.loginButton);
         loginButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                Editable usernameText = ((EditText) findViewById(R.id.username)).getText();
+                Editable usernameText = ((EditText) findViewById(R.id.email)).getText();
```
Comment (pmh192) [inline android/app/src/main/java/com/tourneynizer/tourneynizer/LoginActivity.java]: Change usernameText to emailText

File: android/app/src/main/java/com/tourneynizer/tourneynizer/RegisterActivity.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_register);
+        View registerButton = findViewById(R.id.registerButton);
+        registerButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Editable nameText = ((EditText) findViewById(R.id.name)).getText();
+                Editable usernameText = ((EditText) findViewById(R.id.email)).getText();
```
Comment (pmh192) [inline android/app/src/main/java/com/tourneynizer/tourneynizer/RegisterActivity.java]: usernameText -> emailText

File: android/.idea/markdown-exported-files.xml (hunk 1 above)
Comment (lar-ryan-nick) [reply android/.idea/markdown-exported-files.xml]: I just used the default .gitignore that android studio created. It only emitted some of the .idea directory and kept some so android studio could easily allow someone to "open an existing project." Would you prefer me to remove it completely?

Comment (pmh192) [reply android/.idea/markdown-exported-files.xml]: Cool beans

**Pull request merged**